pyuvdata/uvdata/tests/test_mwa_corr_fits.py

```diff
@@ -620,6 +610,7 @@ def test_remove_dig_gains():
     dig_gains1 = dig_gains1[:, :, np.newaxis, np.newaxis]
     dig_gains2 = dig_gains2[:, :, np.newaxis, np.newaxis]
     uv2.data_array = uv2.data_array / (dig_gains1 * dig_gains2)
+    uv2.history = uv1.history
```

Comment:
It would be good to test that the history is what you expect it to be (i.e. check that what you think you added is making it in) before replacing it with the original history.

Reply:
yes, that makes sense. fixed.